app/views/search_results/index.html.erb

```diff
@@ -1,10 +1,10 @@
 <h2>Search Results</h2>
-<% if @results.medical_care %>
-  <h3>Medical Care results</h3>
+<% if @results.urgent_care %>
+  <h3>Urgent Care results</h3>
   <p>See more results</p>
   <table>
     <tr>
-      <% @results.medical_care[0..2].each do |provider| %>
+      <% @results.urgent_care[0..2].each do |provider| %>
         <td style = "width: 33%; vertical-align: top; padding: 5px;" >
```

Comment:
Looks great!

Not necessary right now, but I'm wondering if there is a more efficient way we could refactor to later on. 
- Could we create an array of all selected categories for the search, and then iterate through each array category in the view to reduce the if/else and number of lines of code (since the view for each section should be the same except for the category name)